src/commands/init.command.ts

```diff
@@ -11,49 +12,70 @@ import { TopLevelCommand } from "./TopLevelCommand.ts";
-      const d: Dir = {
-        name: "",
-        entries: [
-          { name: "README.md", content: readmeMd },
-          { name: ".gitignore", content: gitignore },
-          {
-            name: "kit",
-            entries: [
-              { name: ".gitkeep", content: "" },
-              { name: "README.md", content: kitReadmeMd },
-            ],
-          },
-          { name: "compliance", entries: [{ name: ".gitkeep", content: "" }] },
-          { name: "foundations", entries: [{ name: ".gitkeep", content: "" }] },
-        ],
-      };
-
-      await dir.write(d, "");
-
-      // this is the only place where an absolute path is ok, to show the user unambigously where
-      // the repository is on their file system
-      logger.progress(
-        "generated new collie repository at " + kit.resolvePath(),
-      );
-
-      logger.tipCommand(
-        "add a foundation to configure your cloud platforms",
-        "foundation new",
-      );
-    });
+    .command("init [directory]")
+    .description("initialize a new collie repository in a new directory")
+    .action(
+      async (opts: GlobalCommandOptions, directoryArg: string | undefined) => {
+        const kit = new CollieRepository("./");
+        const logger = new Logger(kit, opts);
+
+        const dir = new DirectoryGenerator(WriteMode.skip, logger);
+
+        const directory: string = directoryArg ||
+          await Input.prompt(
+            `Choose a directory name for your new collie repository`,
+          );
+        const repo: Dir = {
+          name: directory,
+          entries: [],
+        };
+
+        await dir.write(repo, "");
+        Deno.chdir(directory);
+
+        // ensure git is initialized
+        const cliFactory = new CliApiFacadeFactory(kit, logger);
+        const git = cliFactory.buildGit();
+
+        await git.init();
+
+        const d: Dir = {
+          name: "",
+          entries: [
+            { name: "README.md", content: readmeMd },
+            { name: ".gitignore", content: gitignore },
+            {
+              name: "kit",
+              entries: [
+                { name: ".gitkeep", content: "" },
+                { name: "README.md", content: kitReadmeMd },
+              ],
+            },
+            {
+              name: "compliance",
+              entries: [{ name: ".gitkeep", content: "" }],
+            },
+            {
+              name: "foundations",
+              entries: [{ name: ".gitkeep", content: "" }],
+            },
+          ],
+        };
+
+        await dir.write(d, "");
+
+        // this is the only place where an absolute path is ok, to show the user unambigously where
+        // the repository is on their file system
+        logger.progress(
+          "generated new collie repository at " + kit.resolvePath(),
+        );
+
+        logger.tipCommand(
+          `change into the collie repository and add a foundation to configure your cloud platforms`,
```

Comment:
```suggestion
          `Next step: add a foundation to configure your cloud platform(s)`,
```

Reply:
[Deno.chdir](https://deno.land/api@v1.35.3?s=Deno.chdir) changes the working dir for operations while collie is running. After collie exits, I am in the dir I started collie from.